azkaban-common/src/test/java/azkaban/project/ProjectManagerTest.java

```diff
@@ -0,0 +1,162 @@
+    User user = new User("testUser");
+    Project project = createProjectUtil(manager, projectName, projectDescription, user);
+
+    Project fetchedProject = manager.getProject(project.getName());
+    assertProjectMemberEquals(project, fetchedProject);
+    System.out.println("Fetched project by name successfully");
+  }
+
+  @Test
+  public void testFetchProjectById() throws ProjectManagerException {
+    System.out.println("projectManagerTest: testFetchProjectById");
+    ProjectManager manager = createProjectManager();
+    String projectName = "myTestFetchProjectById";
+    String projectDescription = "This is to test fetching a project by id";
+    User user = new User("testUser");
+    Project project = createProjectUtil(manager, projectName, projectDescription, user);
+
+    Project fetchedProject = manager.getProject(project.getId());
+    assertProjectMemberEquals(project, fetchedProject);
+    System.out.println("Fetched project by id successfully");
+  }
+
+  @Test
+  public void testFetchAllProjects() throws ProjectManagerException {
+    System.out.println("projectManagerTest: testFetchAllProjects");
+    ProjectManager manager = createProjectManager();
+    String projectName1 = "myTestFetchAllProjects_1";
+    String projectDescription1 = "This is to test fetching all projects: project_1";
+    User user = new User("testUser");
+    Project project1 = createProjectUtil(manager, projectName1, projectDescription1, user);
+
+    String projectName2 = "myTestFetchAllProjects_2";
+    String projectDescription2 = "This is to test fetching all projects: project_2";
+    Project project2 = createProjectUtil(manager, projectName2, projectDescription2, user);
+
+    List<Project> fetchedProjects = manager.getProjects();
+    Assert.assertTrue("Fetched project_1", fetchedProjects.contains(project1));
+    Assert.assertTrue("Fetched project_2", fetchedProjects.contains(project2));
+    System.out.println("Fetched all projects successfully");
+  }
+
+  private ProjectManager createProjectManager() {
+    Props props = new Props();
+    ProjectLoader loader = new MockProjectLoader(workingDir);
+    return new ProjectManager(loader, props);
+  }
+
+  private Project createProjectUtil(ProjectManager manager, String projectName, String description,
+      User creator) throws ProjectManagerException {
+    Project project = null;
+    try {
+      project = manager.createProject(projectName, description, creator);
+    } catch (ProjectManagerException e) {
+      System.out.println("Creating project failed");
+      Assert.fail(e.getMessage());
+    }
+
+    Assert.assertTrue("Project Id set", project.getId() > -1);
+    Assert.assertEquals("Project name", projectName, project.getName());
+    Assert.assertEquals("Project description", description,
+        project.getDescription());
+    Assert.assertTrue("Project is active", project.isActive());
+    return project;
+  }
+
+  private void assertProjectMemberEquals(Project p1, Project p2) {
```

Comment:
Can this method be reused? 
azkaban.project.JdbcProjectLoaderTest#assertProjectMemberEquals